frontend/src/app/models/page-results.ts

```diff
@@ -0,0 +1,57 @@
+import { Observable, combineLatest, from, of } from 'rxjs';
+import { QueryModel } from './query';
+import { map } from 'rxjs/operators';
+import { SearchService } from '../services';
+import { SearchResults } from './search-results';
+import { Results } from './results';
+import { DocumentPage } from './document-page';
+
+export const RESULTS_PER_PAGE = 20;
```

Comment:
Hm, good one. I think the more conventional way of handling it is to put a control for this next to pagination, so let users decide. But if I was going to work on expanding the results page right now, I would prioritise #1011, #1012, #729, or  #996 - a size control seems low-priority to me. It may bring more clutter than value.

As for adding it to environment settings: that is technically trivial, but then I don't think have a reason to change this between our existing environments. Perhaps we could think more long-term about what we envision for different I-analyzer instances.

Reply:
Stray thought on this: size=20 is a bit trivial, since that is the elasticsearch default. Perhaps if we did not specify this, it could also be configured in the elasticsearch settings? Which would also make it environment-configurable.